client/packages/core/src/index.ts

```diff
@@ -390,7 +390,7 @@ class Auth {
   signInWithIdToken = (params: {
     idToken: string;
     clientName: string;
-    nonce: string | undefined | null;
+    nonce?: string | undefined | null;
```

Comment:
Made `nonce` optional. I thought `undefined` meant you could leave it out, but I guess not.